test/browser_tests/cucumber/features/suites/default/pagination.feature

```diff
@@ -10,3 +10,13 @@ Feature: Pagination
   Scenario: Navigate to the next page
     When I click on the next button
     Then the page url should contain "page=2"
+
+  Scenario: Navigate to the previous page
+    When I click on the next button
+    And I click on the previous button again
+    Then the page url should contain "page=1"
+
+   Scenario: Navigate to nth Page
```

Comment:
This one is indented an extra space.

Reply:
thanks @anselmbradford 
How many ways can I screw this up?